Assets/MRTK/SDK/Features/UX/Prefabs/Pointers/Hololens2ShellGrabPointer.prefab

```diff
@@ -0,0 +1,527 @@
+%YAML 1.1
+%TAG !u! tag:unity3d.com,2011:
+--- !u!1 &1507865967819406
+GameObject:
+  m_ObjectHideFlags: 0
+  m_CorrespondingSourceObject: {fileID: 0}
+  m_PrefabInstance: {fileID: 0}
+  m_PrefabAsset: {fileID: 0}
+  serializedVersion: 6
+  m_Component:
+  - component: {fileID: 4338333115073164}
+  - component: {fileID: 114997244636705472}
+  - component: {fileID: 7376459786343609486}
+  - component: {fileID: 7543848034581082309}
+  m_Layer: 0
+  m_Name: Hololens2ShellGrabPointer
```

Comment:
Would it be possible to use a different / more descriptive name than `Hololens2ShellGrabPointer`? I've seen in the past that some people were confused about `HoloLens2PressableButton` because they thought it could _only_ be used on HL2, when in reality it simply matches the HL2 shell (which itself doesn't tell me much about how this is different from the default). Could we use something like `ConicalGrabPointer` or `ConeBasedGrabPointer`? @julenka